### ISSUE TITLE
Support of eniro maps

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#1286]
Feature request from FB (don't know if it is technically possible):

Is it possible to get Eniro streetmap in C:geo, in Neongeo is with this URL: http//map.eniro.com/geowebcache/service/tms1.0.0/map/[z]/[x]/[iy].png

The small islands, creeks och "beachline" is missing or very deformed, looks fine in Eniro streetmap, even wetlands are missing in google.

Live in sweden

**Comment by ExTechOp** [comment]
As far as I know, the idea of removing tile-based rendering support from mapsforge has not progressed since 2012, so there is really no reason for **not** to merge this into the main c:geo codebase. Could this still happen?

**Comment by SammysHP** [comment]
Merge what? That test branch is four years old (sometimes it is hard to merge something after just one week) and does not exist anymore.

**Comment by ExTechOp** [comment]
Yeah, obviously that instance is long gone, but IMHO the idea of having configurable tile servers should be resurreceted. For example, the National Land Survey of Finland has excellent freely available maps, and there is an unofficial tile server. Are you still there, **rsudev**?

**Comment by SammysHP** [comment]
Sure, he is one of our main developers and is currently working on a mapsforge 0.6 implementation.

**Comment by Souldrinker** [comment]
+1 on this request. When I used Neongeo back in the day I often used the Eniro aerial view which seemed more detailed than google satelites and for example today I just solved a myst that is located in an area where Eniro have paths marked on their maps, but Google/OSM doesn't, so this would be appreciated.

Since NeonGeo stopped being developed like three years ago and it doesn't support hi-res screens in a good way I stopped using that in favor of c:geo. I saw on this thread: http://www.geocaching.se/forum/viewtopic.php?f=66&t=18952, that also CacheSense supports custom map url's, but I haven't used that either for a while since it didn't seem to read in all caches consistently and neither those or the official apps gets close to c:geo in other regards, so having support for custom tile servers in c:geo would be great.
